src/cart/crt_hg.c

```diff
@@ -709,8 +709,7 @@ crt_rpc_handler_common(hg_handle_t hg_hdl)
 		D_GOTO(out, hg_ret = HG_PROTOCOL_ERROR);
 	}
 
-	crt_ctx = (struct crt_context *)HG_Context_get_data(
-			hg_info->context);
+	crt_ctx = (struct crt_context *)HG_Context_get_data(hg_info->context);
```

Comment:
This is returning a void * so should not need a cast.

Reply:
Agree. Will change in case...

Reply:
done